build.assets/Makefile

```diff
@@ -196,6 +198,8 @@ validate-deps: buildbox
 build-in-container: buildbox
 	docker run --rm=true -u $$(id -u) \
 		   -v $(TOP):$(SRCDIR) \
+		   -v $(GOCACHE):/gocache \
```

Comment:
This shares the go build cache directory with the container speeding up rebuilds. Otherwise, it takes 1.5min to 2min for subsequent `make build` invocations (i.e. w/o changes to source code).